### PR TITLE
Release 22.3.0

### PR DESCRIPTION
**Pull request by brendongo**
### Security
- [ ] Changelog has been updated
- [ ] Change has no security implications (otherwise, ping the security team)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,5 @@
-FROM python:3.9-alpine3.15
+# the boltons library is not yet compatible with 3.10 - https://github.com/mahmoud/boltons/issues/294
+FROM python:3.9-alpine
```
[the remark this answers is not in the document]
Comment (ievans) [reply Dockerfile]:
```suggestion
FROM python:3.9-alpine@sha256:eb3ce1f74a5c47b36ded338f9f2fdcdab7846300957c0646cdce3b6c2ec22d2b
```
@brendongo wdyt about doing a proper pin here?

[the remark this answers is not in the document]
Comment (brendongo) [reply Dockerfile]: Wont we miss security patches and the like?

File: src/semgrep_agent.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                To report pull request failures and optionally push findings to semgrep.dev, add a separate job that runs:
+                  $ semgrep ci
+
+                For questions or support, please reach out at https://r2c.dev/slack
+            """
+        ).strip()
+        + "\n\n",
+        file=sys.stderr,
+    )
+    subprocess.run(cmd)
```
[the remark this answers is not in the document]
Comment (brendongo) [reply src/semgrep_agent.py:139]: Basically static

File: Dockerfile (hunk 1 above)
[the remark this answers is not in the document]
Comment (underyx) [reply Dockerfile]: case in point: @ievans's suggestion is already an outdated hash

**Review by underyx — APPROVED** [review]

**Comment by underyx** [comment]
Good release brendon, well done :D

**Pull request merged**